tests/files/tests.py

```diff
@@ -255,6 +258,24 @@ def test_invalid_image(self):
             self.assertEqual(size, (None, None))
 
 
+class GetImageDimensionsFeedBug(unittest.TestCase):
```

Comment:
We can combine this with the previous test class (rename it you like) and move the `@unittest.skipUnless` decorator to the class instead of each method.